packages/yamerl.exs

```diff
@@ -0,0 +1,39 @@
+defmodule Yamerl.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :yamerl,
+     version: "0.3.1-1",
+      
+     deps: deps,
+     description: description,
+     package: package,
+     fetch: fetch]
+  end
+  
+  defp deps do
+    []
+  end
+
+  defp description do
+    """
+    YAML 1.2 parser in pure Erlang
+    """
+  end
+
+  defp package do
+    [files: ~w(src doc include testsuite rebar.config README.md AUTHORS COPYING),
```

Comment:
I guess not, I updated the pull request and exclude everything except for ``~w(src include rebar.config``